_maps/map_files/Mining/Lavaland.dmm

```diff
@@ -830,13 +830,14 @@
 	},
 /turf/closed/wall,
 /area/mine/laborcamp)
+//  shuttle changed to science/aus because the bee shuttle is poopy
 "du" = (
 /obj/docking_port/stationary{
 	dwidth = 6;
 	height = 6;
 	id = "science_outpost";
 	name = "outpost shuttle dock";
-	roundstart_template = /datum/map_template/shuttle/science;
+	roundstart_template = /datum/map_template/shuttle/science/aus;
```

Comment:
why?
just change the docking port variables in the code, don't make an entirely new one *and then* put it in the lavaland map